client/src/containers/ConnectedSidebar/tests/index.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { authenticateUser } from '@onaio/session-reducer';
+import { mount } from 'enzyme';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router';
+import ConnectedSidebar from '..';
+import { store } from '@opensrp/store';
+
+jest.mock('../../../configs/env');
+
+describe('components/ConnectedSidebar', () => {
+  it('renders the ConnectedSidebar component', () => {
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter>
+          <ConnectedSidebar />
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find('aside').props()).toMatchSnapshot();
+    wrapper.unmount();
+  });
+
+  it('renders the ConnectedSidebar when logged in', () => {
+    store.dispatch(
+      authenticateUser(
+        true,
+        {
+          email: 'bob@example.com',
+          name: 'Bobbie',
+          username: 'RobertBaratheon',
+        },
+        {
+          roles: ['ROLE_EDIT_KEYCLOAK_USERS'],
+          username: 'superset-user',
+          user_id: 'cab07278-c77b-4bc7-b154-bcbf01b7d35b',
+        }
+      )
+    );
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter>
+          <ConnectedSidebar />
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find('aside')).toMatchSnapshot();
```

Comment:
Seems like this snapshot did not return anything

Reply:
need to wrap the reactwrapper with enzyme.toJson